src/quest.cpp

```diff
@@ -1138,22 +1182,33 @@ SPECIAL(johnson)
         return TRUE;
       }
       
+      new_q = new_quest(johnson, ch);
+      //Handle out of quests and broken johnsons.
+      if (new_q == 0) {
+          do_say(johnson, "I got nothing for you now. Come back later.", 0, 0);
```

Comment:
Hmmr. This situation is almost impossible to happen. It's redundant checking here. The only way this can happen is if someone deleted the quest from the db in between the character accepting or denying a quest. I'll log it and change the message.